server/helpers/youtube-dl/youtube-dl-cli.ts

```diff
@@ -105,7 +104,7 @@ export class YoutubeDLCLI {
     additionalYoutubeDLArgs?: string[]
   }) {
     let args = options.additionalYoutubeDLArgs || []
-    args = args.concat([ '--merge-output-format', 'mp4', '-f', options.format, '-o', options.output ])
+    args = args.concat([ '--ignore-no-formats-error', '--merge-output-format', 'mp4', '-f', options.format, '-o', options.output ])
```

Comment:
Can you add a comment to explain the purpose of `--ignore-no-formats-error'`?

Reply:
In the code or PR?

The latter I did https://github.com/Chocobozzz/PeerTube/pull/5332#issuecomment-1273356279

Reply:
Yeah not sure, left over by debugging something. Will remove.